[ffmpeg] Build failue

Package: ffmpeg[core,vpx]:x64-osx -> 5.1.2#6

**Host Environment**

- Host: x64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 5787cfa69 2023-04-24 (19 hours ago)

**To Reproduce**

`vcpkg x-set-installed ffmpeg[core,vpx]:x64-osx`

**Failure logs**

```
-- Using cached ffmpeg-ffmpeg-n5.1.2.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/ffmpeg-ffmpeg-n5.1.2.tar.gz
-- Applying patch 0001-create-lib-libraries.patch
-- Applying patch 0002-fix-msvc-link.patch
-- Applying patch 0003-fix-windowsinclude.patch
-- Applying patch 0004-fix-debug-build.patch
-- Applying patch 0005-fix-nasm.patch
-- Applying patch 0006-fix-StaticFeatures.patch
-- Applying patch 0007-fix-lib-naming.patch
-- Applying patch 0009-Fix-fdk-detection.patch
-- Applying patch 0011-Fix-x265-detection.patch
-- Applying patch 0012-Fix-ssl-110-detection.patch
-- Applying patch 0013-define-WINVER.patch
-- Applying patch 0015-Fix-xml2-detection.patch
-- Applying patch 0020-fix-aarch64-libswscale.patch
-- Applying patch 0022-fix-iconv.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/src/n5.1.2-f31542651f.clean
-- Found external ninja('1.11.1').
-- Getting CMake variables for x64-osx
-- Building Options: --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --disable-autodetect --target-os=darwin --enable-appkit --enable-avfoundation --enable-coreimage --enable-audiotoolbox --enable-videotoolbox --cc=cc --cxx=c++ --nm=nm --ar='ar' --disable-ffmpeg --disable-ffplay --disable-ffprobe --disable-avcodec --disable-avdevice --disable-avformat --disable-avfilter --disable-postproc --disable-swresample --disable-swscale --disable-alsa --disable-amf --disable-libaom --disable-libass --disable-avisynth --disable-bzlib --disable-libdav1d --disable-libfdk-aac --disable-libfontconfig --disable-libfreetype --disable-libfribidi --disable-iconv --disable-libilbc --disable-lzma --disable-libmp3lame --disable-libmodplug --disable-cuda --disable-nvenc --disable-nvdec  --disable-cuvid --disable-ffnvcodec --disable-opencl --disable-opengl --disable-libopenh264 --disable-libopenjpeg --disable-libopenmpt --disable-openssl --enable-securetransport --disable-libopus --disable-sdl2 --disable-libsnappy --disable-libsoxr --disable-libspeex --disable-libssh --disable-libtensorflow --disable-libtesseract --disable-libtheora --disable-libvorbis --enable-libvpx --disable-libwebp --disable-libx264 --disable-libx265 --disable-libxml2 --disable-zlib --disable-libsrt --disable-libmfx --enable-cross-compile --pkg-config-flags=--static
-- Building Release Options: --enable-optimizations
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /bin/bash ./build.sh
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel
    Error code: 1
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-config.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-out.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-err.log

Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:596 (vcpkg_execute_required_process)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-config.log</summary>

```
# /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/src/n5.1.2-f31542651f.clean/configure --prefix=/Users/leanderSchulten/git_projekte/vcpkg/packages/ffmpeg_x64-osx --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --disable-autodetect --target-os=darwin --enable-appkit --enable-avfoundation --enable-coreimage --enable-audiotoolbox --enable-videotoolbox --cc=cc --cxx=c++ --nm=nm --ar=ar --disable-ffmpeg --disable-ffplay --disable-ffprobe --disable-avcodec --disable-avdevice --disable-avformat --disable-avfilter --disable-postproc --disable-swresample --disable-swscale --disable-alsa --disable-amf --disable-libaom --disable-libass --disable-avisynth --disable-bzlib --disable-libdav1d --disable-libfdk-aac --disable-libfontconfig --disable-libfreetype --disable-libfribidi --disable-iconv --disable-libilbc --disable-lzma --disable-libmp3lame --disable-libmodplug --disable-cuda --disable-nvenc --disable-nvdec --disable-cuvid --disable-ffnvcodec --disable-opencl --disable-opengl --disable-libopenh264 --disable-libopenjpeg --disable-libopenmpt --disable-openssl --enable-securetransport --disable-libopus --disable-sdl2 --disable-libsnappy --disable-libsoxr --disable-libspeex --disable-libssh --disable-libtensorflow --disable-libtesseract --disable-libtheora --disable-libvorbis --enable-libvpx --disable-libwebp --disable-libx264 --disable-libx265 --disable-libxml2 --disable-zlib --disable-libsrt --disable-libmfx --enable-cross-compile --pkg-config-flags=--static --enable-optimizations --arch=x86_64 --enable-asm --enable-x86asm --extra-cflags=-arch --extra-cflags=x86_64 --extra-ldflags=-arch --extra-ldflags=x86_64
ALL_COMPONENTS='
    
    aac_adtstoasc_bsf
av1_frame_merge_bsf
av1_frame_split_bsf
av1_metadata_bsf
chomp_bsf
dump_extradata_bsf
dca_core_bsf
dv_error_marker_bsf
eac3_core_bsf
extract_extradata_bsf
filter_units_bsf
h264_metadata_bsf
h264_mp4toannexb_bsf
h264_redundant_pps_bsf
hapqa_extract_bsf
hevc_metadata_bsf
hevc_mp4toannexb_bsf
imx_dump_header_bsf
mjpeg2jpeg_bsf
mjpega_dump_header_bsf
mp3_header_decompress_bsf
mpeg2_metadata_bsf
mpeg4_unpack_bframes_bsf
mov2textsub_bsf
noise_bsf
null_bsf
opus_metadata_bsf
pcm_rechunk_bsf
pgs_frame_merge_bsf
prores_metadata_bsf
remove_extradata_bsf
setts_bsf
text2movsub_bsf
trace_headers_bsf
truehd_core_bsf
vp9_metadata_bsf
vp9_raw_reorder_bsf
vp9_superframe_bsf
vp9_superframe_split_bsf
    aasc_decoder
aic_decoder
alias_pix_decoder
agm_decoder
amv_decoder
anm_decoder
ansi_decoder
apng_decoder
arbc_decoder
argo_decoder
asv1_decoder
asv2_decoder
aura_decoder
aura2_decoder
avrp_decoder
avrn_decoder
avs_decoder
avui_decoder
ayuv_decoder
bethsoftvid_decoder
bfi_decoder
bink_decoder
bitpacked_decoder
bmp_decoder
bmv_video_decoder
brender_pix_decoder
c93_decoder
cavs_decoder
cdgraphics_decoder
cdtoons_decoder
cdxl_decoder
cfhd_decoder
cinepak_decoder
clearvideo_decoder
cljr_decoder
cllc_decoder
comfortnoise_decoder
cpia_decoder
cri_decoder
cscd_decoder
cyuv_decoder
dds_decoder
dfa_decoder
...
Skipped 17313 lines
...

BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	#include <stdint.h>
    3	long check_sin(void) { return (long) sin; }
    4	int main(void) { int ret = 0;
    5	 ret |= ((intptr_t)check_sin) & 0xFFFF;
    6	return ret; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc atanf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return atanf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc atan2f 2 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return atan2f(f, g); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc cbrt 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return cbrt(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc cbrtf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return cbrtf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc copysign 2 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return copysign(f, g); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc cosf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return cosf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc erf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return erf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc exp2 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return exp2(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc exp2f 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return exp2f(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc expf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return expf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc hypot 2 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return hypot(f, g); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc isfinite 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return isfinite(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc isinf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return isinf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc isnan 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return isnan(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc ldexpf 2 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return ldexpf(f, g); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc llrint 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return llrint(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc llrintf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return llrintf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc log2 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return log2(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc log2f 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return log2f(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc log10f 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return log10f(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc lrint 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return lrint(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc lrintf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return lrintf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc powf 2 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return powf(f, g); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc rint 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return rint(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc round 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return round(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc roundf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return roundf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc sinf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return sinf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc trunc 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return trunc(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_mathfunc truncf 1 -lm
test_ld cc -lm
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <math.h>
    2	float foo(float f, float g) { return truncf(f); }
    3	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:3:24: warning: cast to smaller integer type 'int' from 'float (*)(float, float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o -lm
check_complexfunc cabs 1
test_ld cc
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <complex.h>
    2	#include <math.h>
    3	float foo(complex float f, complex float g) { return cabs(f * I); }
    4	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:4:24: warning: cast to smaller integer type 'int' from 'float (*)(_Complex float, _Complex float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o
check_complexfunc cexp 1
test_ld cc
test_cc
BEGIN /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
    1	#include <complex.h>
    2	#include <math.h>
    3	float foo(complex float f, complex float g) { return cexp(f * I); }
    4	int main(void){ return (int) foo; }
END /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
cc -I/opt/homebrew/opt/libomp/include -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore -DPIC @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -std=c11 -Werror=partial-availability -fomit-frame-pointer -fPIC -pthread -c -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c
/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.c:4:24: warning: cast to smaller integer type 'int' from 'float (*)(_Complex float, _Complex float)' [-Wpointer-to-int-cast]
int main(void){ return (int) foo; }
                       ^~~~~~~~~
1 warning generated.
cc @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/x64-osx-rel/ldflags.rsp -arch x86_64 -Wl,-dynamic,-search_paths_first -o /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test /var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T//ffconf.0yEsDkgo/test.o
libvpx enabled but no supported decoders found
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-out.log</summary>

```
=== CLEANING FOR x86_64 ===
BUILD_ARCH=x86_64
OPTION_VARIABLE=OPTIONS_x86_64
=== CONFIGURING ===
libvpx enabled but no supported decoders found

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.libera.chat.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-err.log</summary>

```
make: *** No rule to make target `clean'.  Stop.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "mesa",
      "default-features": false,
      "features": [
        "egl"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

This issue could be reproduced with vcpkg 2023-04-07-bedcba5172f5e4b91caac660ab7afe92c27a9895:
````
      END /var/folders/6q/z1_zb3nd7w12wx8_7q3rqw640000gq/T//ffconf.c0q8036L/test.c
15125 cc -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I$(SRC_PATH)/compat/dispatch_semaphore @/Users/vcpkg/Jon/vcpkg/buildtrees/ffmpeg/x64-osx-rel/cflags.rsp -arch x86_64 -s      td=c11 -Werror=partial-availability -E -o /var/folders/6q/z1_zb3nd7w12wx8_7q3rqw640000gq/T//ffconf.c0q8036L/test.o /var/folders/6q/z1_zb3nd7w12wx8_7q3rqw640000gq/T//ffconf.c0q8036L/te      st.c
15126 /var/folders/6q/z1_zb3nd7w12wx8_7q3rqw640000gq/T//ffconf.c0q8036L/test.c:1:10: fatal error: 'sys/brand.h' file not found
15127 #include <sys/brand.h>
15128          ^~~~~~~~~~~~~
15129 1 error generated.
15130 test_cpp_condition sys/version.h defined __DJGPP__
15131 test_cpp
15132 BEGIN /var/folders/6q/z1_zb3nd7w12wx8_7q3rqw640000gq/T//ffconf.c0q8036L/test.c
15133     1 #include <sys/version.h>
15134     2 #if !(defined __DJGPP__)
15135     3 #error "unsatisfied condition: defined __DJGPP__"
15136     4 #endif

````

--- a/ports/ffmpeg/vcpkg.json
+++ b/ports/ffmpeg/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "ffmpeg",
   "version": "5.1.2",
-  "port-version": 8,
+  "port-version": 9,
   "description": [
     "a library to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.",
     "FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created. It supports the most obscure ancient formats up to the cutting edge. No matter if they were designed by some standards committee, the community or a corporation. It is also highly portable: FFmpeg compiles, runs, and passes our testing infrastructure FATE across Linux, Mac OS X, Microsoft Windows, the BSDs, Solaris, etc. under a wide variety of build environments, machine architectures, and configurations."
@@ -648,6 +648,13 @@
     "vpx": {
       "description": "VP8 and VP9 de/encoding via libvpx",
       "dependencies": [
+        {
+          "name": "ffmpeg",
+          "default-features": false,
+          "features": [
+            "avcodec"
+          ]
+        },
         "libvpx"
       ]
     },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2526,7 +2526,7 @@
     },
     "ffmpeg": {
       "baseline": "5.1.2",
-      "port-version": 8
+      "port-version": 9
     },
     "ffnvcodec": {
       "baseline": "11.1.5.0",

--- a/versions/f-/ffmpeg.json
+++ b/versions/f-/ffmpeg.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "dfcd8db452a85e6e6a0ad751dd4610ff1b0e1662",
+      "version": "5.1.2",
+      "port-version": 9
+    },
+    {
       "git-tree": "3ee8d10aa504c74cfef80d47e1901afa3289a544",
       "version": "5.1.2",
       "port-version": 8

